src/main/java/org/broadinstitute/hellbender/utils/Utils.java

```diff
@@ -254,6 +254,46 @@ public static String join(final String separator, final double[] doubles) {
     }
 
     /**
+     * Splits a String using indexOf instead of regex to speed things up.
```

Comment:
do you have data to show that this is faster than `String.split`? If no, use that method. If yes, move this split method to Utils and add unit tests for it.